conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+
+            <p>The intent of the conflict resolution procedures is to ensure that individuals can continue to participate in W3C activities.</p>
+
+            <p>Where it is beneficial to the continued functioning of W3C, a high level summary of results and actions from an investigation may be communicated to the membership, for example if an individual is suspended from a working group or from being a chair in any working group. Otherwise, results and actions should be kept confidential.</p>
+
+            <p>Chairs, ombudspeople, mediators, and investigators can recommend the following actions be taken if resolution is not possible:</p>
+
+            <ol>
+                <li>Corrective action. The individual who had the complaint raised against them will be advised to correct their behaviour going forward, for example not engaging in the same behaviours that resulted in the complaint.</li>
+                <li>Agree to a mediation process. The individual who had the complaint raised against them will be advised to participate in a mediation process, where both parties can agree to certain actions and changes going forward. </li>
+                <li>An official warning. In cases where the actions by an individual are frequent or severe, an individual and their employer (if relevant) will be given an official warning that their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, this may result in the removal or suspension of their W3C participation. There is no time limit for the expiry of the warning.</li>
+                <li>Disciplinary meeting. In cases where the actions by an individual are frequent or severe, and where greater intervention is needed, an individual and a representative from their place of employment (if relevant) will be asked to attend a meeting with W3M. They will be told where their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, they may result in the removal or suspension of their W3C participation.</li>
+                <li>Temporary removal from a working group, or as a chair of a working group or task force facilitator. In cases where the actions by an individual has been frequent and disruptive, or they have failed to adhere to actions agreed through mediation, they will be suspended from participating in any working groups, chair positions, or as a taskforce facilitator for a period of time agreed on by decision makers.</li>
+                <li>Final written warning or suspension. Where individuals have received multiple warnings, across the same or multiple parts of the CoC. The individual and their employer will receive a written warning that any further complaints will result in the removal of their W3C participation. They may also be suspended from W3C for a given length of time.</li>
```

Comment:
I suggest deleting the "Final written warning" item.  How many warnings does someone need?   A prior bullet already says that a warning may be given, and it does not restrict the number of warnings that may be given.  I think it should be up to W3M's discretion to decide how many warnings are reasonable for a given offender, based on the offender's history and the nature of the violations.  A "final warning" is unnecessary.  Let's not tie W3M's hands.